### PR TITLE
MGMT-10933: Delete UI backup route

### PR DESCRIPTION
**Pull request by mareklibra**
The "ztpfw-ui-copy" backup route is created during the course of
domain-change persistence to allow monitoring of progress.

Once all changes are persisted, there is no need to keep the route.
This patch deletes it as a last step of the persist() routine.

### CODE REVIEW

File: ui/backend/src/common/index.ts [hunk 1]
```diff
@@ -2,3 +2,4 @@ export * from './types';
 export * from './validation';
 export * from './resources';
 export * from './domains';
+export * from './constants';
```
Comment (alknopfler) [inline ui/backend/src/common/index.ts:5]: where we're using constants?

Comment (mareklibra) [reply ui/backend/src/common/index.ts:5]: In the context of this PR, it's about the `ZTPFW_NAMESPACE` constant.
Constants exported at this line are shared between backend and frontend.

Comment (alknopfler) [reply ui/backend/src/common/index.ts:5]: ok ok

**Review by alknopfler — APPROVED** [review]
LGTM

**Pull request merged**